locales/en-US/app.l20n

```diff
@@ -68,6 +69,21 @@
 <feedbackUninstallTitle "Thank You!">
 <feedbackUninstallCopy "Your participation in Firefox Test Pilot means a lot! Please check out our other experiments, and stay tuned for more to come!">
 
+<discussNotifyTitle "Just one sec...">
+<discussNotifyMessage "In
```

Comment:
would prefer you did this as two paragraph tags rather than the double <br> in the middle